flow/layers/image_filter_layer_unittests.cc

```diff
@@ -23,7 +24,9 @@ using ImageFilterLayerTest = LayerTest;
 
 #ifndef NDEBUG
 TEST_F(ImageFilterLayerTest, PaintingEmptyLayerDies) {
-  auto layer = std::make_shared<ImageFilterLayer>(sk_sp<SkImageFilter>());
+  auto sk_image_filter = sk_sp<SkImageFilter>();
+  auto layer =
+      std::make_shared<ImageFilterLayer>(DlImageFilter::From(sk_image_filter));
```

Comment:
Similar comment as in the ColorFilter version of this PR. Unless a unit test has a specific reason to use one of the SkImageFilter variants that we don't support, migrate these to using a basic DlImageFilter object instead. I haven't looked through each and every unit test here, but I would be surprised if any of them actually care what kind of filter they are using (except some of their bounds answers might be tailored to the filter they originally chose and may need some adjustment to match a new type of filter.